net/pool/pool_test.go

```diff
@@ -240,16 +357,31 @@ func (d *dialerMock) Name() (name string) {
 	return "net.peerservice"
 }
 
-func newTestPeer(id string) *testPeer {
+func newTestPeer(id string, created time.Time, subConnections int, version uint32) *testPeer {
```

Comment:
make another constructor for testPeer with all that stuff, don't change other tests